### PR TITLE
Add Next.js support

### PR DESCRIPTION
**Pull request by MartinKristof**
Ability to upload source maps after Next.js (>= 5.0.0) build.

### CODE REVIEW

**Comment by jasonbarry** [comment]
This is just what I was looking for! Anything else need to happen for this to land?

**Comment by MartinKristof** [comment]
It is ready to merge I think :)

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-restricted-syntax,global-require */
+const RollbarSourceMapPlugin = require('./RollbarSourceMapPlugin');
+
+export const withRollbar = (nextConfig = {}) => Object.assign({}, nextConfig, {
+  webpack(config, options) {
+    if (!options.defaultLoaders) {
+      throw new Error('This plugin is not compatible with Next.js versions below 5.0.0 https://err.sh/next-plugins/upgrade');
+    }
+
+    const { dev, buildId, isServer } = options;
```
Comment (brandondoran) [inline src/index.js]: What are these `dev` and `isServer` options? They aren't documented.

**Comment by IgorCRD** [comment]
I think Nextjs support may be better addressed by making publicPath argument a function|string.

If string nothing would change.

If function the plugin would call it passing the source local address, this would enable users to apply whatever routing logic they need to and the returned value would be used as publicPath for that file.

This would attend Nextjs routing needs without changing anything for other users, it might as well allow support for other unknown libraries or frameworks that might have similar routing needs.

As this PR looks like its inactive for a long time I would like to give it a try. What you all think?

**Comment by brandondoran** [comment]
@IgorCRD yes, I think `publicPath` argument as a function|string is a great idea that is flexible and backwards compatible. I see you already have #58 so I'll check it out.

**Comment by brandondoran** [comment]
Closing this as the desired end result should be achievable with the `publicPath` arg as function, added in #58

**Pull request closed without merging**